uint/src/uint.rs

```diff
@@ -366,6 +386,21 @@ macro_rules! construct_uint {
 					self.low_u128()
 				}
 			}
+
+			impl $crate::core_::convert::TryFrom<$name> for u128 {
+				type Error = &'static str;
+
+				#[inline]
+				fn try_from(u: $name) -> Result<u128, &'static str> {
+					let $name(arr) = u;
+					for i in 2..$n_words {
```

Comment:
Is this safe if `$n_words` is 2 (or even 1)?

Reply:
[yes](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=d8e0da798a3ff6995ac6be1214829592)